src/controller/middleware/authentication.js

```diff
@@ -2,8 +2,8 @@ exports.auth = (req, res, next) => {
   if (!req.token) {
     next();
   } else if (req.token && req.token.consultant) {
-    return res.render('consProfile');
+    return res.redirect('/consProfile');
```

Comment:
@AymanAlqoqa Just for fun, here's how you could shorten it if you wanted to:
```
if (!req.token) {
  next();
} else {
  return res.redirect( req.token.consultant ? '/consProfile' : '/userProfile' );
}
```

Reply:
okey